consul_kv.go

```diff
@@ -15,12 +16,14 @@ func NewConsulConfigStorage(consulAddress string) (*ConsulConfigStorage, error)
 	return &ConsulConfigStorage{kv: client.KV(), json: json}, nil
 }
 
+// ConsulConfigStorage
 type ConsulConfigStorage struct {
 	config Config
 	kv     *api.KV
 	json   jsoniter.API
 }
 
+// GetConfig is get config from consul kv
 func (cs *ConsulConfigStorage) GetConfig(labName string) (Config, error) {
 	pair, _, err := cs.kv.Get(labName, nil)
 	if err != nil {
```

Comment:
1 put the expression in the method comments, so the user can see them by go-doc
2 It should be the original config (previous config) not the &Config{}, think about getting config multi times, but fail just last time.